socrates/lib/eventstore/RegistrationCommandProcessor.js

```diff
@@ -9,7 +9,8 @@ const misc = beans.get('misc');
 
 class RegistrationCommandProcessor {
 
-  constructor(writeModel) {
+  constructor(url, writeModel) {
+    this._url = url;
     this.writeModel = writeModel;
   }
```

Comment:
hier ist in vielen methoden noch ein if (){ return } else if () { return } else { return } pattern drin. Nicht so elegant. Besser fände ich: if () { return } if () { return } return